src/main/java/org/lightcouch/View.java

```diff
@@ -109,7 +109,11 @@
 	private Boolean includeDocs;
 	private Boolean inclusiveEnd;
 	private Boolean updateSeq;
-	
+
+	/** This will be true initially or if the most recent call to {@link #queryNextPage} or
+	 * {@link #queryPreviousPage} was {@link #queryNextPage}, and false otherwise. */
+	private boolean pagingForwards = true;
```

Comment:
Here too, pagingDirection and either enum or defined forward/backward constants.